jtransc-rt-core/src/com/jtransc/JTranscSystem.java

```diff
@@ -104,7 +104,7 @@ static public void debugger() {
 	@HaxeMethodBody("if (!p0) N.debugger();")
 	@JTranscMethodBodyList({
 		@JTranscMethodBody(target = "js", value = "if (!p0) debugger;"),
-		@JTranscMethodBody(target = "cs", value = "if (!pc) System.Diagnostics.Debugger.Break();"),
+		@JTranscMethodBody(target = "cs", value = "if (!p0) System.Diagnostics.Debugger.Break();"),
```

Comment:
JTranscSystem.TRUE and FALSE failed.  They were translated into 1 and 0, then Mono threw me an error.